components/brave_rewards/resources/img/donate_disabled.svg

```diff
@@ -0,0 +1 @@
.5 3.5 0 1.3.526 3.5 2.5 3.5s2.5-2.2 2.5-3.5c0-1.3-.526-3.5-2.5-3.5zM50 14h2l-1.015-3z" fill="#8B9FE1"/><g><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M21.092 9.476c-.076 0-.151.003-.224.012l-.82.092v7.178h1.232c1.1 0 2.564-.35 2.564-3.368 0-2.689-1.427-3.914-2.752-3.914z" fill="#8B9FE1" mask="url(#b)"/><path d="M79.207.108H12.775C5.719.108 0 5.882 0 13.007c0 7.123 5.719 12.897 12.774 12.897h9.2c-.102-1.116-.034-2.303.406-3.345.479-1.136 1.355-2.001 2.464-2.435a4.534 4.534 0 0 1 1.662-.317c3.629 0 6.247 4.283 6.949 5.933l.07.164h45.683c7.055 0 12.773-5.774 12.773-12.897 0-7.125-5.718-12.899-12.774-12.899zM21.314 18.517c-.306 0-.609-.024-.901-.048-.411-.032-.934-.058-1.256-.02l-1.43.167a.687.687 0 0 1-.755-.577l-.06-.377a.695.695 0 0 1 .61-.798l.686-.067V9.545l-.748-.163a.69.69 0 0 1-.54-.705l.01-.24a.69.69 0 0 1 .793-.654l.227.038c.32.054.65.11.964.11.276 0 .55-.04.865-.088.344-.053.732-.11 1.157-.11 2.96 0 4.797 2.09 4.797 5.458 0 4.634-2.77 5.326-4.42 5.326zm10.056.199c-3.009 0-4.355-2.809-4.355-5.591 0-2.784 1.346-5.592 4.355-5.592 3.008 0 4.354 2.808 4.354 5.592 0 2.782-1.346 5.59-4.354 5.59zM45.44 9.49h-.35v8.334a.687.687 0 0 1-.685.692h-.62a.684.684 0 0 1-.592-.346l-3.821-6.67v5.257h.827c.378 0 .685.31.685.692v.375a.687.687 0 0 1-.685.692H37a.687.687 0 0 1-.684-.692v-.375c0-.382.306-.692.684-.692h.532V9.491H37a.688.688 0 0 1-.684-.69v-.378c0-.381.306-.69.684-.69h1.917c.246 0 .473.132.595.348l3.738 6.609V9.49h-.86a.688.688 0 0 1-.685-.69v-.376c0-.383.307-.692.685-.692h3.05c.378 0 .685.309.685.692V8.8a.688.688 0 0 1-.684.691zm10.676 8.334a.687.687 0 0 1-.684.692H52.66a.687.687 0 0 1-.684-.692v-.375c0-.382.306-.692.684-.692h.548l-.483-1.311h-3.337l-.474 1.311h.57c.377 0 .684.31.684.692v.375a.687.687 0 0 1-.684.692h-2.739a.688.688 0 0 1-.685-.692v-.375c0-.382.306-.692.685-.692h.264l3.378-9.015a.684.684 0 0 1 .636-.447h.003c.282 0 .535.175.638.44l3.5 9.022h.268c.377 0 .684.31.684.692v.375zm9.333-6.453a.688.688 0 0 1-.684.691h-.471a.688.688 0 0 1-.684-.691v-.985c0-.606-.053-.769-.083-.81-.008-.005-.129-.086-.739-.086h-.92v7.266h1.467c.379 0 .685.31.685.693v.375a.688.688 0 0 1-.685.692h-4.776a.688.688 0 0 1-.685-.692v-.375c0-.383.307-.693.685-.693h1.468V9.491h-.92c-.611 0-.732.08-.744.09-.026.037-.078.2-.078.806v.985a.688.688 0 0 1-.684.691h-.471a.688.688 0 0 1-.686-.691V8.423c0-.381.307-.69.686-.69h7.635c.379 0 .684.309.684.69v2.949zm9.629 6.453a.687.687 0 0 1-.684.692h-7.57a.687.687 0 0 1-.684-.692v-.375c0-.382.306-.692.684-.692h.958V9.491h-.958a.688.688 0 0 1-.684-.69v-.378c0-.381.306-.69.684-.69h7.258c.378 0 .684.309.684.69v2.334a.687.687 0 0 1-.684.692h-.472a.687.687 0 0 1-.684-.692v-.154c0-.593-.032-.934-.096-1.014-.017-.015-.15-.098-.84-.098h-2.367v2.307h1.441a.685.685 0 0 1 .673-.564h.471c.379 0 .684.31.684.69v1.671a.687.687 0 0 1-.684.691h-.47a.688.688 0 0 1-.685-.69v-.04h-1.43v3.202h2.662c.953 0 .953 0 .953-1.575v-.04c0-.381.307-.69.685-.69h.47c.38 0 .685.309.685.69v2.682z" fill="#8B9FE1" mask="url(#b)"/></g></g></svg>
```

Comment:
where do we use this?

Reply:
Seems to be not used anymore. Will be deleted.

Reply:
Removed